src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+    private static final int MIN_NUMBER_RANGE = 1;
+    private static final int MAX_NUMBER_RANGE = 45;
+    private static final String RANGE_ERROR_MESSAGE = "숫자는 1~45 사이의 숫자여야한다.";
```

Comment:
```suggestion
    private static final String RANGE_ERROR_MESSAGE = "숫자는 " + MIN + "~" + MAX  + " 사이의 숫자여야한다.";
```

상수를 사용한다면 추후 해당 값이 변경 됐을 때 메시지 변경 없이 처리가 가능할 것 같아요!

Reply:
맞습니다!!